options/options.cpp

```diff
@@ -667,7 +668,13 @@ ProverResult PonoOptions::parse_and_set_options(int argc,
         case HELP:
           // not possible, because handled further above and exits the program
         case ENGINE: engine_ = to_engine(opt.arg); break;
-        case BOUND: bound_ = atoi(opt.arg); break;
+        case BOUND:
+          bound_ = atoi(opt.arg);
+          if (bound_ >= INT_MAX) {
```

Comment:
```suggestion
          if (bound_ == INT_MAX) {
```
Maybe change it to `INT_MAX`, because anything above that is undefined behavior.